.travis.yml

```diff
@@ -0,0 +1,60 @@
+sudo: required
+language: c
+
+matrix:
+  include:
+    - compiler: gcc
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - gcc-4.8
+      env: C_COMPILER=gcc-4.8
+    - compiler: gcc
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - gcc-4.9
+      env: C_COMPILER=gcc-4.9
+    - compiler: gcc
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - gcc-5
+      env: C_COMPILER=gcc-5
+    - compiler: clang
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+            - llvm-toolchain-precise-3.6
+          packages:
+            - clang-3.6
+      env: C_COMPILER=clang-3.6
+    - compiler: clang
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+            - llvm-toolchain-precise-3.7
+          packages:
+            - clang-3.7
+      env: C_COMPILER=clang-3.7
+
+
+before_install:
+  - export CC=$C_COMPILER
+  - wget https://github.com/libcheck/check/files/71408/check-0.10.0.tar.gz
+  - tar xvfz check-0.10.0.tar.gz
+  - cd check-0.10.0 && ./configure && make && sudo make install && cd ..
```

Comment:
I cannot seem to make this work on travis even though it works just fine locally:
```bash
-- Found CHECK: /Users/travis/.local/lib/libcheck.dylib  
-- Found components for CHECK
-- CHECK_ROOT_DIR  = /Users/travis/.local
-- CHECK_INCLUDES  = /Users/travis/.local/include
-- CHECK_LIBRARIES = /Users/travis/.local/lib/libcheck.dylib
-- includes: /Users/travis/.local/include;/Users/travis/build/thinkingfish/pelikan/deps/ccommon/include;/Users/travis/build/thinkingfish/pelikan/_build/ccommon
-- Looking for ck_assert_int_eq
-- Looking for ck_assert_int_eq - not found
```

As you can see, the installed header/lib are found, but `check_symbol_exists` failed regardless. The exact same code and invocation (w/ prefix) works locally for me.

I cannot figure out why this is and it's already taking way too much time to setup travis. I'm creating an issue to optimize the yaml and remove requirement on `sudo` at some point. But I'd like to move forward with the working config for now.